gpflow/covariances/multioutput/kufs.py

```diff
@@ -80,23 +80,42 @@ def _Kuf(
     return tf.stack(Kufs, axis=0)  # [L, M, N]
 
 
-@Kuf.register(
-    (FallbackSeparateIndependentInducingVariables, FallbackSharedIndependentInducingVariables),
-    LinearCoregionalization,
-    object,
-)
-def _Kuf(
+def _fallback_Kuf(
+    kuf_impl,
     inducing_variable: Union[
         SeparateIndependentInducingVariables, SharedIndependentInducingVariables
     ],
     kernel: LinearCoregionalization,
     Xnew: tf.Tensor,
 ):
-    kuf_impl = Kuf.dispatch(type(inducing_variable), SeparateIndependent, object)
     K = tf.transpose(kuf_impl(inducing_variable, kernel, Xnew), [1, 0, 2])  # [M, L, N]
     return K[:, :, :, None] * tf.transpose(kernel.W)[None, :, None, :]  # [M, L, N, P]
 
 
+@Kuf.register(
+    FallbackSeparateIndependentInducingVariables, LinearCoregionalization, object,
+)
+def _Kuf(
+    inducing_variable: SeparateIndependentInducingVariables,
+    kernel: LinearCoregionalization,
+    Xnew: tf.Tensor,
```

Comment:
I'm frustrated that the multipledispatch package doesn't work better together with the type annotations...
this way we've got two places to put it and here they're inconsistent, I believe it should be
```suggestion
@Kuf.register(
    FallbackSeparateIndependentInducingVariables, LinearCoregionalization, object,
)
def _Kuf(
    inducing_variable: FallbackSeparateIndependentInducingVariables,
    kernel: LinearCoregionalization,
    Xnew: tf.Tensor,
```